tests/integration/checkpoint.bats

```diff
@@ -33,8 +33,18 @@ function teardown() {
   [ "$status" -eq 0 ]
   [[ "${output}" == *"running"* ]]
 
+  # pre-dump the running container
+  mkdir parent-dir
+  runc --criu "$CRIU" checkpoint --pre-dump --image-path ./parent-dir test_busybox
+  [ "$status" -eq 0 ]
+
+  # busybox should still be running
+  runc state test_busybox
+  [ "$status" -eq 0 ]
+  [[ "${output}" == *"running"* ]]
+
   # checkpoint the running container
-  runc --criu "$CRIU" checkpoint test_busybox
+  runc --criu "$CRIU" checkpoint --parent-path ./parent-dir test_busybox
```

Comment:
Please make a new test for `--pre-dump`. We shouldn't be replacing old tests with new tests that test different features.